ext/Devel-Peek/t/Peek.t

```diff
@@ -575,7 +575,7 @@ do_test('reference to hash containing Unicode',
       PV = $ADDR "' . $cp200_bytes . '"\\\0 \[UTF8 "\\\x\{200\}"\]
       CUR = 2
       LEN = \\d+
-      COW_REFCNT = 1				# $] < 5.019007
+      COW_REFCNT = 1				# $] < 5.019007 || $] >=5.037007
```

Comment:
Do we know why we stopped doing this in 5.019? That suggests there is a trap we need to be wary of no?